packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-stack-transform.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
we shouldn't disable this rule. If there's a dependency that's not listed in `package.json`, add it